models/app.models.js

```diff
@@ -1 +1,12 @@
-const {} = require("../db/connection");
+const db = require("../db/connection");
+
+exports.readTopics = () => {
+  return db
+    .query(`SELECT * FROM topics;`)
+    .then(({ rows }) => {
+      return rows;
+    })
+    .catch((err) => {
```

Comment:
we dont need to catch and throw the error here - we aren't changing anything so we can allow the promise to just reject and make use of that in the controller